addons/binding/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/model/VVM320SpecialChannels.java

```diff
@@ -0,0 +1,148 @@
+    CH_40782("40782", "EB101 Cpr Frequency Desired F2040 #2", ChannelType.Sensor, ChannelGroup.Compressor,
+            Double.class),
+    CH_44701("44701", "EB101-EP14 Actual Cpr Frequency Outdoor Unit", ChannelType.Sensor, ChannelGroup.Compressor,
+            Double.class),
+    CH_44702("44702", "EB101-EP14 Protection Status Register Outdoor Unit", ChannelType.Sensor, ChannelGroup.Compressor,
+            String.class),
+    CH_44700("44700", "EB101-EP14 Low Pressure Sensor Outdoor Unit", ChannelType.Sensor, ChannelGroup.Compressor,
+            Double.class),
+    // Airsupply
+    CH_40025("40025", "BT20 Exhaust air temp. 1", ChannelType.Sensor, ChannelGroup.Airsupply, Double.class),
+    CH_40026("40026", "BT21 Vented air temp. 1", ChannelType.Sensor, ChannelGroup.Airsupply, Double.class),
+    CH_40075("40075", "BT22 Supply air temp.", ChannelType.Sensor, ChannelGroup.Airsupply, Double.class),
+    CH_40183("40183", "AZ30-BT23 Outdoor temp. ERS", ChannelType.Sensor, ChannelGroup.Airsupply, Double.class),
+    CH_40311("40311", "External ERS accessory GQ2 speed", ChannelType.Sensor, ChannelGroup.Airsupply, Double.class),
+    CH_40312("40312", "External ERS accessory GQ3 speed", ChannelType.Sensor, ChannelGroup.Airsupply, Double.class),
+    CH_40942("40942", "External ERS accessory block status", ChannelType.Sensor, ChannelGroup.Airsupply, String.class),
+
+    /* END */
+    ;
+
+    private final String id;
+    private final String name;
+    private final ChannelType channelType;
+    private final ChannelGroup channelGroup;
+    private final Class<?> javaType;
+
+    /**
+     * Constructor
+     *
+     * @param id
+     * @param name
+     * @param type
+     */
+    VVM320SpecialChannels(String id, String name, ChannelType channelType, ChannelGroup channelGroup,
+            Class<?> javaType) {
+        this.id = id;
+        this.name = name;
+        this.channelType = channelType;
+        this.channelGroup = channelGroup;
+        this.javaType = javaType;
+    }
+
+    public static VVM320SpecialChannels fromId(String id) {
+        for (VVM320SpecialChannels channel : VVM320SpecialChannels.values()) {
+            if (channel.id.equals(id)) {
+                return channel;
+            }
+        }
+        return null;
+    }
+
+    @Override
+    public final String getName() {
+        return name;
+    }
+
+    @Override
+    public final String getId() {
+        return id;
+    }
+
+    @Override
+    public final ChannelType getChannelType() {
+        return channelType;
+    }
+
+    @Override
+    public ChannelGroup getChannelGroup() {
+        return channelGroup;
+    }
+
+    @Override
+    public final Class<?> getJavaType() {
+        return javaType;
+    }
+
+    @Override
+    public String getFQName() {
+        String fQName = getChannelGroup().toString().toLowerCase() + "#" + getId();
```

Comment:
The variable `fQName` is not needed, simply return here.

Reply:
fixed it